.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+ports:
+- port: 3000
+  onOpen: open-preview
+tasks:
+- init: yarn && gp sync-done yarn
```

Comment:
@anudeepreddy what is `gp sync-done yarn`? This file was very understandable before, but now it seems like added maintenance burden. I want to make sure this is added actual developer value without increasing maintenance burden

Reply:
Has anyone else in the Carbon community used Gitpod and can vouch for its benefits?

Reply:
The file opens two different terminals when the workspace is launched for the first time. One to install the dependencies and the other to ask for their contributions. So before we can run `yarn contrib:add` the first terminal should complete installation of all the dependencies. So `gp sync-done yarn` will notify terminals waiting that the job is done. While `gp sync-await yarn` will wait until it is signalled that the job yarn is done.

You can run the workspace yourself by clicking on the link in this PR and in the terminal type `gp` and that will give you the usage of each option.
I will be happy to answer your questions if you have any.

Reply:
Would it be possible to just leave the contributions stuff as is? That would simplify this file a lot.

For context, we already have a bot that handles it (@all-contributors)

Reply:
Yeah we can do that 🙂

Reply:
I made the changes. Now I hope this PR is ready to be merged